tox.ini

```diff
@@ -2,8 +2,6 @@
 downloadcache = {distshare}
 args_are_paths = false
 envlist =
-    py{35,36,37,38,39}-dj22
```

Comment:
Shouldn't we wait until April 2022 to remove Django 2.2?

Reply:
No, once Django 4.0 is out, it's time to drop support for all versions older than the most recent LTS. Otherwise, it could require ugly shims to add 4.0 support (which removes a lot of deprecated things). This is outlined in https://www.djangoproject.com/weblog/2015/jun/25/roadmap/.

Reply:
Thanks for the explanation!